src/graph/SimpleAuthenticator.h

```diff
@@ -8,22 +8,29 @@
 #define GRAPH_SIMPLEAUTHENTICATOR_H_
 
 #include "base/Base.h"
+#include "base/MD5Utils.h"
 #include "graph/Authenticator.h"
+#include "graph/GraphFlags.h"
 
 namespace nebula {
 namespace graph {
 
 class SimpleAuthenticator final : public Authenticator {
 public:
+    explicit SimpleAuthenticator(meta::MetaClient *metaClient) {
+        metaClient_ = metaClient;
+    }
     bool MUST_USE_RESULT auth(const std::string &user,
                               const std::string &password) override {
-        if (user == "user" && password == "password") {
+        if (!FLAGS_security_authorization_enable) {
```

Comment:
Good point, I want to set this variable only when start the `nebula-graphd` process. Therefore, we need to pay attention to this issue in future development and reject dynamically setting this flag.

Reply:
consider the situation: someone initially write the config file, and set this flag enabled. And later change this flag is disabled. And (after some options) later reset as enabled. It may cause some bad situation (data corruption or leaking). Add  a TODO.